core-web/libs/edit-content/src/lib/fields/dot-edit-content-host-folder-field/dot-edit-content-host-folder-field.component.html

```diff
@@ -7,11 +7,12 @@
     [attr.aria-labelledby]="'field-' + field.variable"
     [virtualScroll]="true"
     [virtualScrollItemSize]="50"
+    [scrollHeight]="'450px'"
```

Comment:
Should we add a test to check that `scrollHeight` and `virtualScrollOptions.style.height` are equal?